tests/formulagrader/test_intervalgrader.py

```diff
@@ -0,0 +1,79 @@
+"""
+Tests for FormulaGrader and NumericalGrader
+"""
+from __future__ import print_function, division, absolute_import
+
+from pytest import raises
+from voluptuous import Error
+
+from mitxgraders import IntervalGrader, NumericalGrader, FormulaGrader, StringGrader
+from mitxgraders.exceptions import ConfigError, InvalidInput
+
+def test_usage():
```

Comment:
I totally believe that it works, but do you want to add a test using FormulaGrader with variables?

Reply:
Done.